doc/users-guide/modules/ROOT/pages/faq/custom-tls-certificates.adoc

```diff
@@ -0,0 +1,28 @@
+= Custom TLS certificates
+:navtitle: Custom TLS certificates
+:description: When connecting to TLS with custom TLS certificates, users need to add them to the truststore of the IDE.
+
+{description}
+
+== Symptom
+
+When rendering an AsciiDoc document which for example connects to a Kroki diagram server that presents a self-signed certificate, the output contains an error similar to the following:
+
+====
+org.jruby.exceptions.StandardError: (SSLError) asciidoctor: FAILED: ...: Failed to load AsciiDoc document - certificate verify failed
+====
+
+== Cause: Untrusted certificate authority
+
+This happens if the remote server presents a certificate which is not trusted.
+Usually this happens if it uses a self-signed certificate or an unknown/private root certificate authority.
+
+== How do I fix this?
+
+Add the self-signed certificate or the matching root certificate authority to the IntelliJ's `cacerts` file.
+
```

Comment:
cacerts file is located here: INTELLIJ_HOME/jbr/lib/security/cacerts.

This solution requires write access to cacerts, which may be an issue on company maintained/protected laptops.

The easiest solution to me is to simply remove `kroki-fetch-diagram: true` from antora-playbook.yaml